recipes/recipes_emscripten/gtest/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -ex
+
+# Build and install dynamic library
+if [ ! -d "build" ]; then
+  mkdir build
+fi
+cd build
+
+cmake ${CMAKE_ARGS} \
+  -GNinja \
+  -DCMAKE_INSTALL_PREFIX=$PREFIX \
+  -DCMAKE_CXX_STANDARD=17 \
+  -DBUILD_SHARED_LIBS=ON \
```

Comment:
When you go through the build logs, you'll see warnings like this:
```
ADD_LIBRARY called with SHARED option but the target platform does not
support dynamic linking.  Building a STATIC library instead.  This may lead
to problems.
```
Prefer using static libraries. So change this to:
```
-DBUILD_SHARED_LIBS=OFF \
```